src/layers/conv.jl

```diff
@@ -393,3 +393,43 @@ function Base.show(io::IO, l::EdgeConv)
     print(io, ", aggr=", l.aggr)
     print(io, ")")
 end
+
+
+"""
+    GINConv([graph, ] nn, eps, train_eps
```

Comment:
```suggestion
    GINConv(fg,] nn, eps, train_eps)
```
Also need to expand the docstring. 
`eps` and `train_eps` could be keyword args

Reply:
I removed train_eps for now, but I expanded the docstring.